src/server/modules/post/post.router.ts

```diff
@@ -205,7 +206,7 @@ export const postRouter = router({
       const post = await ctx.prisma.post.update({
         where: { id },
         data: {
-          deletedAt: new Date(),
+          deletedAt: getCurrSgtTime(),
```

Comment:
May not want to use SGT time for DBs, feels ahhhh dates in DBs are a separate headache….

Reply:
it'll still get stored in UTC!